lib/PTEVal.ml

```diff
@@ -38,20 +45,25 @@ and pp_af ok = pp_int_field ok "af" (fun p -> p.af)
 and pp_db ok = pp_int_field ok "db" (fun p -> p.db)
 and pp_dbm ok = pp_int_field ok "dbm" (fun p -> p.dbm)
 and pp_el0 ok = pp_int_field ok "el0" (fun p -> p.el0)
+and pp_attrs ok = pp_field ok (fun a -> sprintf "attrs:(%s)" (String.concat ", " a))
+                    attrs_eq (fun p -> p.attrs)
```

Comment:
Wel `ok=false` meaning do not print when equal to default. Something more involved is called for if we want arch-dependant defaults. This may be an extra parameter to the printing functions... Or significant refactoring to get arch-dependant pteval's.

Reply:
A simpler solution would be to consider that pteval are used for AArch64 only and to delay refactoring until another architecture implements virtual memory.

Reply:
I happy to add a functor, I've uploaded a version where the AArch64 defaults are hardcoded  but if you prefer the  clean solution, I'm happy to give it a go.